ipatests/test_integration/test_nfs.py

```diff
@@ -236,22 +233,61 @@ def test_automount_location(self):
         # maybe re-use m1.group(0) if it exists.
         assert m1 is None
 
+        # https://pagure.io/freeipa/issue/7918
+        # check whether idmapd.conf was setup using the IPA domain
+        automntclt.run_command([
+            "grep", "Domain = %s" % self.master.domain.name, "/etc/idmapd.conf"
+        ])
```

Comment:
So you're relying on raising an exception?

Reply:
If it fails will there still be cleanup?

Reply:
It should break execution if the return code is not 0.